apps/frontend/src/app/pages/dashboard/maps/upload-form/map-upload-form.component.ts

```diff
@@ -248,10 +247,11 @@ export class MapUploadFormComponent implements OnInit, AfterViewInit {
     let mapCreated = false;
     let mapID = -1;
 
-    const mapObject: CreateMap = {
+    const mapObject: any = {
+      // CreateMap = {
       name: this.name.value,
       fileName: this.name.value, // TODO: Proper filename field
-      type: this.type.value,
+      // type: this.type.value,
```

Comment:
Commented?

Reply:
Probably getting rid of it, still slightly unsure if we want to store a `primaryMode` on the map or something. Ultimately this file is changing drastically in the future, don't mind leaving it a bit messy until I come back to it a couple weeks